open-build-service-api/src/lib.rs

```diff
@@ -48,16 +49,18 @@ impl std::fmt::Display for ApiError {
 
 type Result<T> = std::result::Result<T, Error>;
 
-#[derive(Clone, Copy, Deserialize, Debug, Eq, PartialEq)]
+#[derive(Clone, Copy, Deserialize, Debug, Eq, PartialEq, Display)]
 #[serde(rename_all = "lowercase")]
+#[strum(serialize_all = "snake_case")]
 pub enum RebuildMode {
     Transitive,
     Direct,
     Local,
 }
 
-#[derive(Clone, Copy, Deserialize, Debug, Eq, PartialEq)]
+#[derive(Clone, Copy, Deserialize, Debug, Eq, PartialEq, Display)]
 #[serde(rename_all = "lowercase")]
+#[strum(serialize_all = "snake_case")]
```

Comment:
serde lowercase  but strum snake_case ? (thought it won't matter, it's still odd)